spec/models/invenio_rdm_record_converter_spec.rb

```diff
@@ -207,18 +208,19 @@
               "identifier": "PNB-14-96-a",
               "scheme": "other"
             },
-            {
-              "identifier": "3956501241",
-              "scheme": "isbn"
-            },
             {
               "identifier": "PNB-15-10",
               "scheme": "other"
```

Comment:
Hmmm, is it possible to get a repeat "other" in the export? I didn't encounter a record with a repeat "other" identifier scheme in the latest export, so we are good for now. But if there is a repeat, we would probably be in the same position... If the "other" scheme is never repeated (and it seems like it so far), we don't have to do anything about it.

Reply:
The only original identifiers that are mapping to other right now are those that have PNB in them. I think that only a single PNB identifier is applied to each file since they seem to follow a pattern which is meant to identify which brief it's from and what page it is (or something along those lines hehe)